lib/sdf-server/src/service/v2/module/contribute.rs

```diff
@@ -83,6 +93,7 @@ pub async fn contribute(
             "pkg_hash": response.latest_hash,
         }),
     );
+    ctx.commit().await?;
```

Comment:
I'm guessing we want this now because `ctx.write_audit_log().await?` will flush its events on `ctx.commit().await?`?

Reply:
There was already a commit in the route! The bad GitHub diff got me too :/ took me a bit to find it

Reply:
There's also an equivalent to "publish immediately" for audit logs to skip the rebaser _I think_. I had it at some point... can't remember if it's still there.